### PR TITLE
Added exercises for 2.4

### PR DESCRIPTION
**Pull request by bbjornstad**
I added exercises for section 2.4, 7 from the Evans text and a further 3 from OpenStax on projections to supplement.

### CODE REVIEW

File: src/chapters/chapter2.tex [hunk 1]
```diff
@@ -1040,8 +1040,77 @@ \subsection{Projection}
 compute a projection without computing cosine of anything!
 
 \begin{exercises}
+	\begin{enumerate}
+		\item   In each case determine if the given pair of vectors is perpendicular.
```
Comment (siefkenj) [inline src/chapters/chapter2.tex]: perp -> orthogonal

File: src/chapters/chapter2.tex [hunk 2]
```diff
@@ -1040,8 +1040,77 @@ \subsection{Projection}
...
+			regular tetrahedron and a Carbon atom at its center.  Choose as the vertices of
+			this tetrahedron the points $(0,0,0)$, $(1,1,0)$, $(1,0,1)$, and $(0,1,1)$.
+			\begin{enumerate}
+				\item Find the angle between two edges of the tetrahedron.
+				\item Find the bond angle between two Carbon--Hydrogen bonds.
+			\end{enumerate}
+		\item   An inclined plane makes an angle of $30$ degrees with the horizontal.  Use
+			vectors and the dot product to find the projection of the gravitational
+			acceleration vector $-g\yhat$ along a unit vector pointing down the inclined
+			plane.
```
Comment (siefkenj) [inline src/chapters/chapter2.tex:1064]: down -> along

File: src/chapters/chapter2.tex [hunk 3]
```diff
@@ -1040,8 +1040,77 @@ \subsection{Projection}
...
+		\item   Use the dot product to determine if the points $P(3,1,2)$, $Q(-1,0,2)$, and
+			$R(11, 3, 2)$ are collinear.
+
+	\end{enumerate}
 \end{exercises}
 
+\begin{openstaxexercises}
+	\begin{enumerate}
+		\item	For the following exercises, the vectors $\vec{u}$ and $\vec{v}$ are given. Find
+			the projection $w = \Proj_{\vec{u}}\vec{v}$ of vector $\vec{v}$ onto vector
```
Comment (siefkenj) [inline src/chapters/chapter2.tex]: \vec w

File: src/chapters/chapter2.tex [hunk 4]
```diff
@@ -1040,8 +1040,77 @@ \subsection{Projection}
...
+			$\vec{u}$. Express your answer in component form.
+			\begin{enumerate}
+				\item	$\vec{u} = 5\xhat + 2\yhat$, $\vec{v} = 2\xhat + 3\yhat$.
+				\item	$\vec{u} = (−4, 7)$, $\vec{v} = (3, 5)$	
+				\item	$\vec{u} = 3\xhat + 2\zhat$, $\vec{v} = 2\yhat + 4\zhat$
+				\item	$\vec{u} = (4, 4, 0)$, $\vec{v} = (0, 4, 1)$
+			\end{enumerate}
+		\item	Consider the vectors $\vec{u} = 4\xhat − 3\yhat$ and $\vec{v} = 3\xhat + 2\yhat$
+			\begin{enumerate}
+				\item	Find the component form of vector $\vec{w} =
```
Comment (siefkenj) [inline src/chapters/chapter2.tex]: "find the components of" instead of "component form"

**Comment by bbjornstad** [comment]
I have revised my branch to reflect the suggested changes.

**Pull request merged**